legacy_packages/sdk/src/evm/common/modular/compatibleExtensions.ts

```diff
@@ -0,0 +1,90 @@
+import { providers, utils } from "ethers";
+import {
+  coreContractAbi,
+  extensionContractAbi,
+} from "../../constants/thirdweb-features";
+import { hasDuplicates } from "../utils";
+import { getChainByChainIdAsync, getChainRPC } from "@thirdweb-dev/chains";
+
+export async function compatibleExtensions(
+  coreBytecode: string,
+  extensionBytecodes: string[],
+  chainId: number,
+): Promise<boolean> {
+  const chain = await getChainByChainIdAsync(chainId);
+  const rpcUrl = getChainRPC(chain);
+  const jsonRpcProvider = new providers.JsonRpcProvider(rpcUrl);
+  const addr = "0x0000000000000000000000000000000000000124";
```

Comment:
what his this hardcoded address here?

Reply:
This is an arbitrary address. Here it's using eth_call state override to mock deploy on this address and call functions without actually deploying contract.

Reply:
goes over my head but sounds good!